src/view/drag-handle/sensor/create-touch-sensor.js

```diff
@@ -23,12 +23,75 @@ type TouchWithForce = Touch & {
   force: number
 }
 
+type SafariHack = {|
+  start: () => void,
+  stop: () => void,
+|}
+
 export const timeForLongPress: number = 150;
 export const forcePressThreshold: number = 0.15;
 const touchStartMarshal: EventMarshal = createEventMarshal();
-
 const noop = (): void => { };
 
+// Safari does not allow event.preventDefault() in dynamically added handlers
+// So we add an always listening event handler to get around this :(
+// webkit bug: https://bugs.webkit.org/show_bug.cgi?id=184250
+const safariHack: SafariHack = (() => {
+  const shouldBlock: boolean = (() => {
+    // Do nothing when server side rendering
+    if (typeof window === 'undefined') {
+      return false;
+    }
+
+    const isUsingSafari11: RegExp = /AppleWebKit.*Version\/11/g;
+
+    // Not using Safari 11
+    if (!isUsingSafari11.test(window.navigator.userAgent)) {
+      return false;
+    }
+
+    // Using Safari 11 with no touch support - no point adding the touch listeners
+    if (!('ontouchstart' in window)) {
+      return false;
+    }
+
+    return true;
+  })();
+
+  if (!shouldBlock || typeof window === 'undefined') {
```

Comment:
Nit: The `typeof window` part of the conditional won't get hit since it's being checked within `shouldBlock`.